Instance name: /mnt/c/Users/ORG/CLionProjects/uMV-FSTSP/Murray_Chu_2015_test_data/FSTSP/FSTSP_10_customer_problems/FSTSP_10_customer_problems
terminate called after throwing an instance of 'std::invalid_argument'
  what():  stoi
